library/std/src/os/unix/net/ancillary.rs

```diff
@@ -193,6 +219,11 @@ pub struct SocketCred(libc::ucred);
 #[derive(Clone)]
 pub struct SocketCred(libc::sockcred);
 
+#[cfg(freebsd13)]
```

Comment:
You also need to change the `#[cfg` statement on line 206 or else `SocketCred` will be defined twice.